### PR TITLE
Add 5 tips for nd_player_tips

### PR DESCRIPTION
**Pull request by gallomimia**
5 starting tips

### CODE REVIEW

File: updater/nd_player_tips/nd_player_tips.phrases.txt [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
+	"Stealth Capture"
+	{
+		"en"		"You Cannot Capture Resources While Stealthed."
+	}
+	
+	"Resource Timer Reset"
+	{
+		"en"		"Leaving the Resource or Using Special Abilities Resets the Capture Timer! Stay on the point to help your team."
+	}
+
```
Comment (adam-ma-roberts) [inline updater/nd_player_tips/nd_player_tips.phrases.txt]: "Second grenade with in 2 seconds"
	{
		"en"		"Multiple grenades have no increased damage on target, target different areas or wait till the first has exploded. "
	}

Comment (stickz) [reply updater/nd_player_tips/nd_player_tips.phrases.txt]: How about this: 
"Multiple grenades deal no damage to the target. Throw in different areas or wait till the first has exploded. "

File: updater/nd_player_tips/nd_player_tips.phrases.txt [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
-	"Example Tip 2"
+	"Grenade Spam"
 	{
-		"en"		"Example description"
+		"en"		"Grenades Destroy Each Other! Only Throw One at a Time. You can Cook your grenades to shorten the Fuse!"
 	}
+	
+	"Lockdown Capture"
+	{
+		"en"		"You Cannot Capture Resources While in Lockdown Mode."
```
Comment (adam-ma-roberts) [inline updater/nd_player_tips/nd_player_tips.phrases.txt]: "You can not capture Resources whilst in Lockdown Mode, disable Lockdown (press 'F') to capture this resource."

File: updater/nd_player_tips/nd_player_tips.phrases.txt [hunk 3]
```diff
@@ -1,17 +1,34 @@
...
+	}
+	
+	"Stealth Capture"
+	{
+		"en"		"You Cannot Capture Resources While Stealthed."
```
Comment (adam-ma-roberts) [inline updater/nd_player_tips/nd_player_tips.phrases.txt]: "You can not capture Resources whilst in Stealth mode, disable Stealth (press 'F') to capture this resource ."

File: updater/nd_player_tips/nd_player_tips.phrases.txt [hunk 4]
```diff
@@ -1,17 +1,34 @@
...
+	}
+	
+	"Resource Timer Reset"
+	{
+		"en"		"Leaving the Resource or Using Special Abilities Resets the Capture Timer! Stay on the point to help your team."
```
Comment (adam-ma-roberts) [inline updater/nd_player_tips/nd_player_tips.phrases.txt]: "Leaving the Resource or using Special Abilities resets the capture timer! Stay on the point to help your team." 

Also this is different for Prime and other resources, prime does a complete reset, but other resources fade backwards gradually. Need to check if this is the same for secondaries and tertiary points.

File: updater/nd_player_tips/nd_player_tips.phrases.txt [hunk 5]
```diff
@@ -1,17 +1,34 @@
...
+	{
+		"en"		"You Cannot Capture Resources While in Lockdown Mode."
+	}
+	
+	"Stealth Capture"
+	{
+		"en"		"You Cannot Capture Resources While Stealthed."
+	}
+	
+	"Resource Timer Reset"
```
Comment (adam-ma-roberts) [inline updater/nd_player_tips/nd_player_tips.phrases.txt]: "Prime Resource Timer Reset"

File: updater/nd_player_tips/nd_player_tips.phrases.txt [hunk 6]
```diff
@@ -1,17 +1,34 @@
...
+	{
+		"en"		"Leaving the Resource or Using Special Abilities Resets the Capture Timer! Stay on the point to help your team."
+	}
+	
+
```
[the remark this answers is not in the document]
Comment (adam-ma-roberts) [reply updater/nd_player_tips/nd_player_tips.phrases.txt]: Also need to mention some units have Thermal Vision, get a few newbies asking how ... Imazu killed them. (thought better of pretending it was me, not very believable)

**Pull request merged**